.github/workflows/tests.yml

```diff
@@ -14,11 +14,11 @@ jobs:
       matrix:
         os: [ubuntu-latest, macos-13, windows-2019]
         architecture: [x64]
-        geos: [3.8.4, 3.9.5, 3.10.6, 3.11.4, 3.12.2, 3.13.0beta1, main]
+        geos: [3.9.0, 3.9.5, 3.10.6, 3.11.4, 3.12.2, 3.13.0beta1, main]
```

Comment:
Is it needed to test this 4-year old version that had bug fixes applied afterwards? (as now we need to add special cases to our tests for failures that have already been fixed in GEOS also for 3.9)

I am personally fine with our general practice of only testing the latest GEOS releases

Reply:
Generally yes, this matrix should only have the latest GEOS versions. But this is temporarily a placeholder to make the GHA matrix work for the oldest Python version. This will be removed when the older Python version is removed from the matrix. But also, libgeos 3.9.0 is [still in oldstable bullseye debian](https://packages.debian.org/bullseye/libgeos-dev).

Reply:
> But this is temporarily a placeholder to make the GHA matrix work for the oldest Python version

Yes, but for that purpose we could also just use 3.9.5 for that? (so testing that version for both Python versions)

Reply:
> But also, libgeos 3.9.0 is [still in oldstable bullseye debian](https://packages.debian.org/bullseye/libgeos-dev).

This is for https://github.com/shapely/shapely/issues/2119, but if we start looking at those kind things, one could also say that GEOS 3.8 is still in Ubuntu 20.04 LTS ..

Reply:
As I thought, repeating GEOS 3.9.5 skips over Py3.8 testing. The matrix needs unique numbers to work, hence why I picked one. However, this isn't a big deal as I expect this to be gone soon when the Python versions are assessed.